tests/integration_test/10x_notification_dss_test.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Per an offline discussion, I agree that we can do this later!

Reply:
I tried creating a test 10x bundle in staging, but there is an issue replicating it in gcp so there is currently not a bundle we can use. I think we can add it in later since we are running this test in the int environment at the moment!